pkg/consts/enrichment_injection.go

```diff
@@ -5,6 +5,7 @@ const (
 	EnrichmentInjectedEnv        = "METADATA_ENRICHMENT_INJECTED"
 	EnrichmentWorkloadKindEnv    = "DT_WORKLOAD_KIND"
 	EnrichmentWorkloadNameEnv    = "DT_WORKLOAD_NAME"
+	EnrichmentClusterNameEnv     = "DT_CLUSTER_NAME"
```

Comment:
By testing it I saw that the value is essentially the same as the KubernetesClusterMEID in the dynakube status that I introduced with the monitoredEntitiyReconciler. Should we align the names as both field and env var have the same value or should we stick with as it is?

Reply:
thought about it and I am still conflicted about it, because I don't like the inconsistency
but I like `KubernetesClusterMEID` name more, as its more exact, but in the context of the enrichment that does not work

Reply:
I mean if it does not work I think we should stick with your approach, was just a thought that came across 👍

Reply:
"does no work" == the property is just name differently, and we have no control over it, so I can't rename it 😅 

And the name of `...MEID` works better in other parts of our code